packages/bpk-animate-height/src/AnimateHeight.js

```diff
@@ -45,19 +45,12 @@ class AnimateHeight extends Component {
     };
   }
 
-  componentDidMount() {
-    if (this.contentElement && this.props.height === 0) {
-      this.contentElement.style.display = 'none';
```

Comment:
was this tested with screen readers? As I believe elements with `display: none` are hidden from screen readers so removing this might mean it's now read?